app/res/layout/screen_login.xml

```diff
@@ -51,60 +59,91 @@
-                    android:orientation="vertical"
-                    android:paddingBottom="4dp"
-                    android:paddingLeft="@dimen/login_box_margins"
-                    android:paddingRight="@dimen/login_box_margins" >
-
-            <TextView
-                android:id="@+id/text_username"
-                android:layout_width="fill_parent"
-                android:layout_height="wrap_content"
-                android:gravity="left"
-                android:text="@+id/TextView02" />
-
-            <EditText
-                android:id="@+id/edit_username"
-                android:layout_width="fill_parent"
-                android:layout_height="wrap_content"
-                android:inputType="text"
-                android:nextFocusDown="@+id/edit_password" android:contentDescription="Username">
-
-                <requestFocus />
-            </EditText>
 
-            <TextView
-                android:id="@+id/text_password"
-                android:layout_width="fill_parent"
-                android:layout_height="wrap_content"
-                android:gravity="left|center_horizontal"
-                android:text="@+id/TextView03" />
+            <!--<ScrollView-->
+            <!--android:layout_width="match_parent"-->
+            <!--android:layout_height="wrap_content">-->
 
-            <EditText
-                android:id="@+id/edit_password"
-                android:layout_width="fill_parent"
+            <LinearLayout
+                android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_marginBottom="-4dp"
-                android:inputType="textPassword"
-                android:nextFocusDown="@+id/checkserver"
-                android:nextFocusUp="@+id/edit_username" />
+                android:orientation="vertical"
+                android:paddingBottom="4dp"
+                android:paddingLeft="@dimen/login_box_margins"
+                android:paddingRight="@dimen/login_box_margins" >
 
-            <TextView
-                android:id="@+id/screen_login_bad_password"
-                android:layout_width="fill_parent"
-                android:layout_height="wrap_content"
-                android:text="Incorrect Username or Password" android:background="@drawable/bubble_danger" android:paddingTop="2dp" android:paddingBottom="2dp" android:textColor="@color/red" android:textStyle="bold" android:visibility="gone" android:layout_marginLeft="2dp" android:paddingLeft="5dp" android:paddingRight="5dp" android:layout_marginRight="2dp" android:gravity="center"/>
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="Welcome back! Please log in."
+                    android:textColor="@color/cc_neutral_color"
+                    android:layout_gravity="center"
+                    android:layout_margin="16dp"

```

Comment:
This should be in a common styleset, right? 16dp margins and medium text size for a textview seem like they are likely to be applied broadly?